app/assets/stylesheets/content/_forms.lsg

```diff
@@ -178,6 +178,16 @@
         The more, the better
       </div>
     </div>
+    <div class="form--field">
+      <label class="form--label">Telephone:</label>
+      <div class="form--field-container">
+        <div class="form--field-affix icon-context icon-phone">
+        </div>
+        <div class="form--text-field-container">
+          <input type="number" class="form--text-field">
```

Comment:
Number is really not apropriate for a phone number (you wouldn't store a phone number as integer in a database, would you?)

First I thought we can ignore this, since it is only the style guide. But then I realized that the style guide is the place where you copy your dom elements to paste them elsewhere, so it might be better to fix it...